test/unit/widgets/peoplePicker.test.ts

```diff
@@ -388,3 +590,49 @@ describe('GroupBuilder.setGroupName', () => {
     expect(groupBuilder.setGroupName()).toMatchInlineSnapshot('Promise {}')
   })
 })
+
+describe('Person', () => {
+  it('exists', () => {
+    expect(new Person()).toBeInstanceOf(Person)
+  })
+})
+describe('Person.render', () => {
+  it('exists', () => {
+    expect(new Person().render).toBeInstanceOf(Function)
+  })
+  it('runs', () => {
```

Comment:
Try to be a bit more specific about what the test does - "runs" is a bit to generic.